arcgis-ios-sdk-samples/Utility network/Perform valve isolation trace/PerformValveIsolationTraceViewController.swift

```diff
@@ -104,6 +91,31 @@ class PerformValveIsolationTraceViewController: UIViewController {
         return parameters
     }
     
+    /// Load the service geodatabase and initialize the layers.
+    func loadServiceGeodatabase() {
+        serviceGeodatabase = AGSServiceGeodatabase(url: featureServiceURL)
+        serviceGeodatabase.load { [ weak self ] error in
+            guard let self = self else { return }
+            // The gas line layer ./3 and gas device layer ./0 are created from the service geodatabase.
+            self.layers = {
+                let layerIDs = [
```

Comment:
Same here: https://github.com/Esri/arcgis-runtime-samples-ios/pull/858#discussion_r418925647 we may just directly create an array with hardcoded values.